cypress-tests/cypress/e2e/ConnectorUtils/Trustpay.js

```diff
@@ -14,6 +14,58 @@ const successfulThreeDSTestCardDetails = {
     "card_cvc": "737",
 };
 
+const idealBankRedirectDetails = {
+    "bank_redirect": {
+        "ideal": {
+            "billing_details": {
+                "billing_name": "John Doe"
+            },
+            "bank_name": "ing",
+            "preferred_language": "en",
+            "country": "NL"
+        }
+    }
+}
+
+const giropayBankRedirectDetails = {
+    "bank_redirect": {
+        "giropay": {
+            "billing_details": {
+                "billing_name": "John Doe"
+            },
+            "bank_name": "ing",
+            "preferred_language": "en",
+            "country": "DE"
+        }
+    }
+}
+
+const sofortBankRedirectDetails = {
+    "bank_redirect": {
+        "sofort": {
+            "billing_details": {
+                "billing_name": "John Doe"
+            },
+            "bank_name": "hypo_noe_lb_fur_niederosterreich_u_wien",
+            "preferred_language": "en",
+            "country": "NL"
+        }
+    }
+}
+
+const epsBankRedirectDetails = {
+    "bank_redirect": {
+        "eps": {
+            "billing_details": {
+                "billing_name": "John Doe"
+            },
+            "bank_name": "hypo_noe_lb_fur_niederosterreich_u_wien",
+            "preferred_language": "en",
```

Comment:
Could you check with bank redirect pm data, and give appropriate fields , and please remove unnecessary ones